plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -455,7 +455,7 @@ def _onCloudFlowPossible(self) -> None:
             self._start_cloud_flow_message = Message(
                 text = i18n_catalog.i18nc("@info:status", "Send and monitor print jobs from anywhere using your Ultimaker account."),
                 lifetime = 0,
-                image_source = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "resources", "svg",
+                image_source = "file:///" + os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "resources", "svg",
```

Comment:
Wouldn't QUrl().toLocalFile() be a better way to make a URL? Mixing the os.path path (which is backward-slash-based on windows) with a file:/// url is bound to result in issues.

Reply:
`toLocalFile()` goes the wrong direction, but getting QUrl to do it is indeed a good idea. `QUrl.fromLocalFile()` looks like the one we want.